src/components/ActiveTodos/ActiveTodos.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+type Props = {
+  activeItems: number;
+};
+
+export const ActiveTodos: React.FC<Props> = ({ activeItems }) => {
+  return (
+    <span className="todo-count" data-cy="todosCounter">
+      {`${activeItems} items left`}
```

Comment:
also i found an issue with a strange amount of items
![image](https://user-images.githubusercontent.com/88807331/194544493-60aaabf9-accd-405d-84ad-8fb8195c7bb6.png)